modules/services/analytics/src/main/java/au/com/shiftyjelly/pocketcasts/analytics/FirebaseAnalyticsTracker.kt

```diff
@@ -23,6 +23,9 @@ class FirebaseAnalyticsTracker @Inject constructor(
             AnalyticsEvent.PLUS_PROMOTION_NOT_NOW_BUTTON_TAPPED,
             AnalyticsEvent.PLAYBACK_PLAY,
             AnalyticsEvent.PODCAST_SUBSCRIBED,
+            AnalyticsEvent.BANNER_AD_IMPRESSION,
+            AnalyticsEvent.BANNER_AD_TAPPED,
```

Comment:
❓ Shouldn't these be tracked as well?

Reply:
Sorry, I'm not sure I understand. As they are in the list, they will be tracked.